packages/manager/apps/dedicated/client/app/ip/components/list/block/move/ip-ip-block-move.controller.js

```diff
@@ -77,13 +113,72 @@ export default /* @ngInject */ (
     }
   };
 
+  $scope.moveIpv6Action = (serviceName, ipBlock) => {
+    Vrack.addIpv6(serviceName, ipBlock)
+      .then(({ data }) => {
+        $scope.watingTask(data.id, serviceName, () => {
+          $scope.loading.save = false;
+          $scope.resetAction();
+        });
+      })
+      .catch((err) => {
+        Alerter.alertFromSWS(
+          $translate.instant('ip_table_manage_move_ipblock_failure', {
+            t0: $scope.data.ipBlock.ipBlock,
+            t1: $scope.model.serviceName.service,
+          }),
+          err,
+        );
+        $scope.resetAction();
+      });
+  };
```

Comment:
```suggestion
  $scope.moveIpv6Action = (serviceName, ipBlock) => {
    Vrack.addIpv6(serviceName, ipBlock)
      .then(({ data }) => {
        $scope.watingTask(data.id, serviceName, () => {
          $scope.loading.save = false;
        });
      })
      .catch((err) => {
        Alerter.alertFromSWS(
          $translate.instant('ip_table_manage_move_ipblock_failure', {
            t0: $scope.data.ipBlock.ipBlock,
            t1: $scope.model.serviceName.service,
          }),
          err,
        );
      }) .finally(() => {
         $scope.resetAction();
      });;
  };
```

Reply:
the first  $scope.resetAction(); is called when $scope.watingTask will be finished so we can't move it in finally
if I will move it in finally it will be called before all task will be finished